debug.ts

```diff
@@ -33,6 +33,11 @@ let client: BlueLinky;
 let vehicle;
 const { username, password, pin } = config;
 
+let { useInfo } = config;

```

Comment:
You could always

```ts
const { username, password, pin, useInfo = false } = config;
```

Reply:
Can't use const, but can let! Didn't know I could do this, thanks! :)

Reply:
Wait, I lied, fixed.